python/entry_points/py_console_script_binary.bzl

```diff
@@ -13,12 +13,14 @@
 # limitations under the License.
 
 """
-Creates an executable (a non-test binary) for console_script entry points.
-
-```{include} /_includes/py_console_script_binary.md
-```
+Please use `py_console_script_binary` from `//python:pip_bzl` instead.
```

Comment:
Nit, I think this should be pip.bzl instead of pip_bzl.

Maybe it would be great to write the full load statement to help users?

Reply:
SGTM. Will FIXUP.